### DISCUSSION TITLE
Welcome to ZendeskApi_v2 Discussions!

### DISCUSSION THREAD

**Discussion by mozts2005** [mozts2005/ZendeskApi_v2#490] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ruannbotes** [comment]
Hi @mozts2005,

I’ve recently started using your NuGet package and noticed that while it includes endpoints for `TicketFields`, it lacks support for `UserFields`. I’ve implemented the missing functionality and would love to contribute back. However, I’m encountering issues while running the unit and integration tests.

I noticed that your `TestBase` class pulls settings from `appsettings.json` using the following code:

```csharp
var configuration = new ConfigurationBuilder()
    .SetBasePath(TestContext.CurrentContext.TestDirectory)
    .AddJsonFile("appsettings.json", optional: true)
    .AddUserSecrets("411e2606-2274-427d-ad1e-89c0d4bc9f5a", true)
    .AddEnvironmentVariables()
    .Build();

Admin = configuration.GetSection("admin").Get<UserSettings>();
Organization = configuration.GetSection("organization").Get<OrganizationSettings>();
Api = new ZendeskApi(Organization.SiteURL, Admin.Email, Admin.Password);
```
The main branch currently only includes the organization section in the appsettings.json file, causing tests to fail due to the missing admin section. I assume I might need to set up a Zendesk test account and populate these values to get the integration tests working, but I’m unsure if that’s the intended approach.

Could you clarify what’s needed to successfully run the integration tests? Once I have everything working, I’d be happy to submit a pull request for your review.

Thanks for your time, and I look forward to your feedback!

**Reply by mozts2005** [reply]
The repository automation will run the full test. You just need to make sure the test you added can run and pass. 


One of the task I am working is a test clean up so that all dependencies are created in the test. But I have other things that are needed more attention now.

**Reply by mozts2005** [reply]
Here is the admin section for your review. as you will need to add one on your system

```json  
"admin" : {
    "id": 281513402,
    "email": "csharpzendeskapi1234@gmail.com",
    "password": "**********",
    "apiToken": "*********"
  }
````
